app/views/partials/_properties.html.erb

```diff
@@ -5,6 +5,8 @@
       controller:  "<%= controller_name %>"
     },
     text: {
+      default_option: "<%= MetadataPresenter::DefaultText[:option] %>",
+      default_option_hint: "<%= MetadataPresenter::DefaultText[:option_hint] %>",
```

Comment:
`option_hint` doesn't exist yet 🙈   will add it to the presenter

Reply:
It is done here https://github.com/ministryofjustice/fb-metadata-presenter/pull/106/files#diff-4409634b16cce2a09de91d2c9f41ef625824f53167276fc33d95b9394f740c4cR8  it will work 🤞